app.py

```diff
@@ -37,48 +37,57 @@ def exportBag():
     workspace = req_query_params['workspace']
     namespace = req_query_params['namespace']
     auth = req_headers['Authorization']
-    # Dict to collect status codes (or booleans) for successful operations.
-    statusCode = {
-        'status codes': {
-            'workspace exists': '',
-            'stood up workspace': '',
-            'participant upload': '',
-            'sample upload': ''
-        }
-    }
-    manifest_io = ManifestIO(data, url, workspace, namespace, auth, statusCode)
+
+    manifest_io = ManifestIO(data, url, workspace, namespace, auth)
     # Check whether the workspace exists in this namespace, otherwise create it.
-    r = manifest_io.workspace_exists()
-    if r.status_code != 200 and r.status_code != 404:
+    r_check = manifest_io.workspace_exists()
+    if r_check.status_code != 200 and r_check.status_code != 404:
```

Comment:
This is minor and you don't have to change, but I would have checked for the common case first. It's also easier to read positive conditions of negative conditions. Something like:

```
if r_check.status_code == 200:
  ...
elif r.check_status_code == 404:
  ..
else
  # Handle everything else